rskj-core/src/main/java/co/rsk/peg/PegUtils.java

```diff
@@ -0,0 +1,38 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.bitcoinj.core.TransactionOutput;
+import co.rsk.bitcoinj.wallet.Wallet;
+import co.rsk.peg.bitcoin.BitcoinUtils;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+
+import java.util.List;
+import java.util.Optional;
+
+public class PegUtils {
+    private PegUtils() { }
+
+    public static PegTxType getTransactionType(
+        ActivationConfig.ForBlock activations,
+        BridgeStorageProvider provider,
+        Wallet liveFederationsWallet,
```

Comment:
```suggestion
        Wallet liveFederationsWallet or activeFederationWallet,
```

Reply:
By live federations, I mean retiring and active federations. This term is widely used in this project as well as others like fed-changer.